### DISCUSSION TITLE
Website: Typo in Roadmap page

### DISCUSSION THREAD

**Discussion by clicktodev** [zed-industries/zed#12191] [Feature Requests]
It says `implement Zed in Zed` which seems like a typo
![image](https://github.com/zed-industries/zed/assets/34947993/66966c6b-ebc8-4ea3-be42-378d805bd30a)

**Comment by SomeoneToIgnore** [comment]
It is not a typo, but rather one large aspect of Zed: collaboration features.
The idea is to do things like https://github.com/zed-industries/zed/pull/11035#issuecomment-2080174432 and help people implement missing/fix existing Zed things using Zed as a collaborative editor.

**Reply by clicktodev** [reply]
Then shouldn't it be `implement Zed using Zed` instead of `in`
